ocpp/v201/enums.py

```diff
@@ -1350,3 +1351,1347 @@ class SecurityEventType(str, Enum):
+    fan_speed = "FanSpeed"
+
+
+class BayOccupancySensorVariableName(str, Enum):
+    """
+    Variable names where the component type is BayOccupancySensor
+    See PhysicalComponentName for referenced physical component
+    """
+
+    active = "Active"
+    enabled = "Enabled"
+    percent = "Percent"
+
+
+class BeaconLightingVariableName(str, Enum):
+    """
+    Variable names where the component type is BeaconLighting
+    See PhysicalComponentName for referenced physical component
+    """
+
+    active = "Active"
+    color = "Color"
+    enabled = "Enabled"
+    enabled_set = "Enabled(Set)"
+    percent = "Percent"
+    percent_set = "Percent(Set)"
+    power = "Power"
+    problem = "Problem"
+
+
+class CableBreakawaySensorVariableName(str, Enum):
+    """
+    Variable names where the component type is CableBreakawaySensor
+    See PhysicalComponentName for referenced physical component
+    """
+
+    active = "Active"
+    enabled = "Enabled"
+    tripped = "Tripped"
+
+
+class CaseAccessSensorVariableName(str, Enum):
+    """
+    Variable names where the component type is CaseAccessSensor
+    See PhysicalComponentName for referenced physical component
+    """
+
+    active = "Active"
+    enabled = "Enabled"
+    enabled_set = "Enabled(Set)"
+    problem = "Problem"
+    tripped = "Tripped"
+
+
+class ChargingStationVariableName(str, Enum):
+    """
+    Variable names where the component type is ChargingStation
+    See PhysicalComponentName for referenced physical component
+    """
+
+    ac_current = "ACCurrent"
+    ac_voltage = "ACVoltage"
+    ac_voltage_max_limit = "ACVoltage(MaxLimit)"
+    allow_new_sessions_pending_firmware_update = "AllowNewSessionsPendingFirmwareUpdate"
+    available = "Available"
+    availability_state = "AvailabilityState"
+    charge_protocol = "ChargeProtocol"
+    current_imbalance = "CurrentImbalance"
+    ec_variant = "ECVariant"
+    enabled = "Enabled"
+    identity = "Identity"
+    model = "Model"
+    operating_times = "OperatingTimes"
+    overload = "Overload"
+    phase_rotation = "PhaseRotation"
+    power = "Power"
+    power_max_limit = "Power(MaxLimit)"
+    problem = "Problem"
+    serial_number = "SerialNumber"
+    supply_phases = "SupplyPhases"
+    supply_phases_max_limit = "SupplyPhases(MaxLimit)"
+    tripped = "Tripped"
+    vendor_name = "VendorName"
+    voltage_imbalance = "VoltageImbalance"
+
+
+class ChargingStatusIndicatorVariableName(str, Enum):
+    """
+    Variable names where the component type is ChargingStatusIndicator
+    See PhysicalComponentName for referenced physical component
+    """
+
+    active = "Active"
+    color = "Color"
+
+
+class ConnectedEVVariableName(str, Enum):
+    """
+    Variable names where the component type is ConnectedEV
+    See PhysicalComponentName for referenced physical component
+    """
+
+    protocol_agreed = "ProtocolAgreed"
+    protocol_supported_by_ev = "ProtocolSupportedByEV"
+    vehicle_id = "VehicleID"
```

Comment:
This is iffy. It's written `VehicleId` in the appendices PDF Section 3.2.13 and in variables.csv. Also all the other IDs like `EvseId` use the lower case `d`. But in `dm_components_vars.csv:116:ConnectedEV;VehicleID` it uses upper case...

Reply:
Nice, catch this should be now corrected